source/codegen/metadata/nirfmxnr/__init__.py

```diff
@@ -0,0 +1,11 @@
+from .functions import functions
```

Comment:
^^^ Did you use the lib from the installer and/or my branch?

Reply:
Yeah, I just took it from your branch before I had it set up and updated it in this commit: https://github.com/ni/grpc-device/pull/466/commits/7ab5c264c54642546becdb0a95cda5260f2df266.